src/nevermined/Assets.ts

```diff
@@ -92,6 +121,8 @@ export function getRoyaltyAttributes(nvm: Nevermined, kind: RoyaltyKind, amount:
  * Assets submodule of Nevermined.
  */
 export class Assets extends Instantiable {
+
+    static DEFAULT_REGISTRATION_ACTIVITY_ID = SignatureUtils.hash('0x1')
```

Comment:
do we want thist activity id to be `1` or the hash of one?

Reply:
The hash. Changed in any case to:
```
    static DEFAULT_REGISTRATION_ACTIVITY_ID = SignatureUtils.hash('AssetRegistration')
```